xbmc/events/BaseEvent.cpp

```diff
@@ -20,6 +20,9 @@
 
 #include "BaseEvent.h"
 #include "guilib/LocalizeStrings.h"
+#include "utils/StringUtils.h"
+
+unsigned int CBaseEvent::s_nextIndex = 0;
```

Comment:
Is this something you forgot to remove? Can't see it used and can't see a reason for it either, increasing the granularity of the timestamp should be enough